From e6624a94dbace6fbaa424980741efa368bc61803 Mon Sep 17 00:00:00 2001
In-Reply-To: <20190222105658.26831-1-avarab@gmail.com>
References: <20190222105658.26831-1-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Fri, 22 Feb 2019 14:18:16 +0100
Subject: [PATCH v2 4/6] Makefile: Move *_LIBS assignment into its own section
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Now the only other non-program assignment in the previous list is
PTHREAD_CFLAGS, which'll be moved elsewhere in a follow-up chang.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 Makefile | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 0870fd4651..59674ce9d7 100644
--- a/Makefile
+++ b/Makefile
@@ -572,7 +572,6 @@ TCLTK_PATH = wish
 XGETTEXT = xgettext
 MSGFMT = msgfmt
 CURL_CONFIG = curl-config
-PTHREAD_LIBS = -lpthread
 PTHREAD_CFLAGS =
 GCOV = gcov
 STRIP = strip
@@ -580,6 +579,9 @@ SPATCH = spatch
 
 export TCL_PATH TCLTK_PATH
 
+# Set our default LIBS variables
+PTHREAD_LIBS = -lpthread
+
 # user customisation variable for 'sparse' target
 SPARSE_FLAGS ?=
 # internal/platform customisation variable for 'sparse'
-- 
2.21.0.rc2.1.g2d5e20a900.dirty

